Lib/test/test_subprocess.py

```diff
@@ -2165,16 +2163,6 @@ def raise_runtime_error():
             subprocess.call([sys.executable, '-c', ''],
                             preexec_fn=lambda: None)
             self.assertTrue(gc.isenabled(), "Popen left gc disabled.")
-
-            gc.disable = raise_runtime_error
-            self.assertRaises(RuntimeError, subprocess.Popen,
-                              [sys.executable, '-c', ''],
-                              preexec_fn=lambda: None)
-
-            del gc.isenabled  # force an AttributeError
-            self.assertRaises(AttributeError, subprocess.Popen,
-                              [sys.executable, '-c', ''],
-                              preexec_fn=lambda: None)
         finally:
             gc.disable = orig_gc_disable
             gc.isenabled = orig_gc_isenabled
```

Comment:
saving and restoring these two with the try: finally: seems no longer necessary now that they're not being monkeypatched out to cause errors.  (harmless, but anyone reading this will wonder why...)

Reply:
Oh, you're right: I created https://github.com/python/cpython/pull/25709